source/gravity_model/radial.cc

```diff
@@ -178,6 +183,159 @@ namespace aspect
+      const double x = p[0];
+      const double y = p[1];
+      const double z = p[2];
+
+      const double C1 = std::cos( 2. * b_NSR / R1 / year_in_seconds * t);
+      const double C2 = std::sin( 2. * b_NSR / R1 / year_in_seconds * t);
+
+      const double dTstar_over_dx = 1. / 6. * ( 2. * x );
+      const double dT0_over_dx    = 1. / 2. * ( 2. * C1 * x - 2. * C2 * y );
+
+      const double dTstar_over_dy = 1. / 6. * ( 2. * y );
+      const double dT0_over_dy    = 1. / 2. * ( -2. * C1 * y - 2. * C2 * x );
+
+      const double dTstar_over_dz = 1. / 6. * ( -4. * z );
+      const double dT0_over_dz    = 0;
+
+      const double G = aspect::constants::big_g;
+      const double T_factor = 3. * G * M_p / 2. / a_s / a_s / a_s;
+
+      const Tensor<1,dim> tidal_gravity = T_factor *
+                                          ( (dTstar_over_dx + dT0_over_dx) * e_x
+                                            + (dTstar_over_dy + dT0_over_dy) * e_y
+                                            + (dTstar_over_dz + dT0_over_dz) * e_z);
+
+      return RadialLinear<dim>::gravity_vector(p) + tidal_gravity;
+    }
+
+
+    template <int dim>
+    void
+    RadialLinearWithTidalPotential<dim>::declare_parameters (ParameterHandler &prm)
+    {
+      RadialLinear<dim>::declare_parameters (prm);
+
+      prm.enter_subsection("Gravity model");
+      {
+        prm.enter_subsection("Radial linear with tidal potnetial");
+        {
+          prm.declare_entry ("Mass of planet", "1.898e27",
+                             Patterns::Double (),
+                             "Mass of satellte's host planet. "
+                             "Default value is for Europa, therefore, mass of Jupiter. "
+                             "Units is $kg$.");
+          prm.declare_entry ("Semimajor axis of satellite", "670900000",
+                             Patterns::Double (),
+                             "Length of semimajor axis of satellite. "
+                             "Default value is for Europa's semimajor axis"
+                             "Units is $m$.");
+          prm.declare_entry ("Rate of nonsynchronous rotation", "1000",
+                             Patterns::Double (),
+                             "Rate of nonsynchronous rotation (NSR). "
+                             "This will be converted to angular rate. "
+                             "Units is $m/year$");
+        }
+        prm.leave_subsection ();
+      }
+      prm.leave_subsection ();
+    }
+
+
+    template <int dim>
+    void
+    RadialLinearWithTidalPotential<dim>::parse_parameters (ParameterHandler &prm)
+    {
+      AssertThrow (dim==3, ExcMessage ("The 'radial linear with tidal potential' gravity model "
+                                       "can only be used in 3D."));
+      this->RadialLinear<dim>::parse_parameters (prm);
+
+      prm.enter_subsection("Gravity model");
+      {
+        prm.enter_subsection("Radial linear with tidal potnetial");
```

Comment:
typo in the section name. Also just use "Radial with tidal potential". That allows extending the plugin later.